script/QA/QA_13.txt

```diff
@@ -143,6 +143,7 @@ and you're a goner!
 // Funny cat later says 'I already told him that!' re: 'attack swiftly and retreat',
 // so I guess that's the meaning of 気まぐれ that's intended here. Not sure that any direct TLs of
 // whim, caprice, fickle, etc. are a great match here, so some liberty taken with an expression instead.
+// group check pr: add something about button masher also memes
```

Comment:
Wrong line, this is about the 'rampant character' - 
> That enemy is a real button masher, it feels like he's%{n}
> constantly unleashing new special moves.

Reply:
damn I fumbled that one